test/TestPublicResolver.js

```diff
@@ -419,6 +419,113 @@ contract('PublicResolver', function (accounts) {
         });
     });
 
+    describe('dns', async () => {
+
+        it('permits setting name by owner', async () => {
+            // a.eth. 3600 IN A 1.2.3.4
+            const arec = '016103657468000001000100000e10000401020304';
+            // b.eth. 3600 IN A 2.3.4.5
+            const b1rec = '016203657468000001000100000e10000402030405';
+            // b.eth. 3600 IN A 3.4.5.6
+            const b2rec = '016203657468000001000100000e10000403040506';
```

Comment:
Can you add a test with >1 record with the same name and rtype?

Reply:
Unless I misunderstand the request that is what the highlighted section does (two records for `b.eth`, both `A` records).

Reply:
My mistake.